components/more-less/more-less.js

```diff
@@ -234,13 +237,27 @@ class MoreLess extends LocalizeCoreElement(LitElement) {
 		this.expanded = true;
 	}
 
-	__focusIn() {
+	__focusIn(e) {
 		if (this.inactive || this.expanded) {
 			return;
 		}
 
-		this.__expand();
-		this.__autoExpanded = true;
+		const target = e.composedPath()[0];
+
+		if (isSafari) {
+			target.scrollIntoViewIfNeeded();
+		}
+
+		if (this.__content.scrollTop) {
+			this.__content.scrollTo({ top: 0, behavior: 'instant' });
```

Comment:
We scroll the content div back to the top instantly. This fixes the expand transition when using the keyboard to focus. It currently jumps to pull the active element into view within the collapsed content area, then expands it, reversing the intended animation from revealing the bottom to revealing the top.